build.gradle

```diff
@@ -184,6 +184,12 @@ dependencyManagement {
     dependencySet(group: 'org.mockito', version: '3.2.4') {
       entry 'mockito-core'
     }
+    dependencySet(group: 'org.apache.tomcat.embed', version: '9.0.30') {
+      entry 'tomcat-embed-core'
+    }
+    dependencySet(group: 'org.apache.tomcat.embed', version: '9.0.30') {
+      entry 'tomcat-embed-websocket'
+    }
```

Comment:
Can be squashed under same `dependencySet` as group and version match

Reply:
done